kube-runtime/src/reflector/object_ref.rs

```diff
@@ -167,3 +187,50 @@ impl<K: Resource> From<ObjectRef<K>> for ObjectRef<ErasedResource> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{ErasedResource, ObjectRef};
+    use k8s_openapi::api::{
+        apps::v1::Deployment,
+        core::v1::{Node, Pod},
+    };
+
+    #[test]
+    fn display_should_follow_expected_format() {
+        assert_eq!(
+            format!("{}", ObjectRef::<Pod>::new("my-pod").within("my-namespace")),
+            "Pod.v1./my-pod.my-namespace"
```

Comment:
this format is a bit annoying, normally core/v1 is special cased, but it's perhaps not worth it

Reply:
Yeah, I don't like it either, but it's the same as what `kubectl` uses (try `kubectl get Pod.v1./foo`). Sadly `kubectl` doesn't support the `.namespace` syntax, but at least the idea is that you can copy/paste the rest straight into it.

Reply:
That's a good enough reason to not special-case it here at least!